src/main/java/com/celements/cells/ICellWriter.java

```diff
@@ -32,29 +32,33 @@
    * @deprecated since 2.82 instead use openLevel(String, List<CellAttribute>)
    */
   @Deprecated
-  public void openLevel(@Nullable String tagName, @Nullable String idname,
-      @Nullable String cssClasses, @Nullable String cssStyles);
+  void openLevel(@Nullable String tagName, @Nullable String idname, @Nullable String cssClasses,
+      @Nullable String cssStyles);
 
   /**
    * @deprecated since 2.82 instead use openLevel(List<CellAttribute>)
    */
   @Deprecated
-  public void openLevel(String idname, String cssClasses, String cssStyles);
+  void openLevel(String idname, String cssClasses, String cssStyles);
 
-  public void closeLevel();
+  void closeLevel();
 
-  public void clear();
+  void clear();
 
-  public String getAsString();
```

Comment:
Do not remove. Mark deprecated and add deprecated notice.

Reply:
it's not removed but moved, see line 59